pkg/feeds/putter.go

```diff
@@ -57,21 +57,7 @@ func (u *Putter) Put(ctx context.Context, i Index, at int64, payload []byte) err
 }
 
 func toChunk(at uint64, payload []byte) (swarm.Chunk, error) {
-	hasher := bmtpool.Get()
-	defer bmtpool.Put(hasher)
-
-	ts := make([]byte, 8)
+	ts := make([]byte, swarm.SpanSize)
```

Comment:
note that `ts` is a timestamp here, so it is indeed `8` bytes because of the fact it is a Uint64, but I would prefer to initialize the variable with a hardcoded `8` bytes since using the `const` in `swarm` package might mislead the reader to think this is somehow related to the chunk span.